src/vm/corhost.cpp

```diff
@@ -1302,6 +1302,8 @@ HRESULT CorHost2::ExecuteAssembly(DWORD dwAppDomainId,
 
     HRESULT hr = S_OK;
 
+    AppDomain *pCurDomain = SystemDomain::GetCurrentDomain();
```

Comment:
Thanks. It is left over from a previous state of this change where the code on line 1329 was just after this declaration.
Fixed.